src/iop/censorize.c

```diff
@@ -345,8 +345,8 @@ int process_cl(struct dt_iop_module_t *self, dt_dev_pixelpipe_iop_t *piece, cl_m
   err = dt_opencl_enqueue_copy_image(devid, dev_out, dev_tmp, origin, origin, region);
   if(err != CL_SUCCESS) goto error;
 
-  sizes[0] = ROUNDUPWD(width);
-  sizes[1] = ROUNDUPWD(height);
+  sizes[0] = ROUNDUPDWD(width, devid);
+  sizes[1] = ROUNDUPDWD(height, devid);
```

Comment:
Shouldn't this be ROUNDUPDHT? Maybe a bug in previous implementation BTW.

Reply:
yes, should be fixed. And yes again, good to proofread all of them. Same comment for all other problems following ...